pokemongo_bot/__init__.py

```diff
@@ -212,6 +212,8 @@ def work_on_cell(self, cell, position):
                     if hack_chain > 10:
                         #print('need a rest')
                         break
+                    if self.config.mode == "poke":
```

Comment:
The problem is that both workers are not giving any informations back to the bot so they need to be refactored. This quickfix should work until that is being done.